### PR TITLE
[INJI-702]:Renamed Scan as Share in the Bottom tab NavBar

### PR DESCRIPTION
**Pull request by Anil-kumar-Majji**
**[INJI-702](https://mosip.atlassian.net/browse/INJI-702)** : Rename Scan as Share in the Navbar.

[**Respective Screenshot**](https://github.com/mosip/inji/assets/106086523/9da4dd66-bd80-460b-90b5-16a7bfc0f677)

[INJI-702]: https://mosip.atlassian.net/browse/INJI-702?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: locales/ara.json [hunk 1]
```diff
@@ -356,6 +356,7 @@
   },
   "MainLayout": {
     "home": "المنزل",
+    "share": "يشارك",
     "scan": "مسح",
```
Comment (KiruthikaJeyashankar) [inline locales/ara.json:360]: Can we remove unused locales?

Comment (KiruthikaJeyashankar) [reply locales/ara.json:360]: It is used in QR scan screen header

**Review by Anil-kumar-Majji — COMMENTED** [review]
Everything looks good. If is there any required changes, please make comments.

**Review by KiruthikaJeyashankar — APPROVED** [review]

**Review by KiruthikaJeyashankar — APPROVED** [review]

**Review by vharsh — APPROVED** [review]
lgtm

**Pull request merged**